time series notebooks feedback from pete (#1959)
deephaven/deephaven-core: demo/web/src/main/notebooks/A. Real-time table ops/A3 Do time series and relational joins.md

Let's join these tables using an `aj` to get a single table with all of our information.

```python
joined_data_aj = daily_data_0.aj(daily_data_1, "Timestamp", "Number1 = Number, Character1 = Character, Boolean1 = Boolean")
```

> Deephaven supports another type of as of join, an `raj`. For a `raj`, the values in the right table are matched to the closest values in the left table without going under the left value. For example, if the right table contains a value `5` and the left table contains values `4` and `6`, the right table's `5` will be matched on the left table's `4`.

Reviewer: ```suggestion
Deephaven supports another type of as-of-join, a reverse as-of-join. Using `raj`, the values in the right table are matched to the closest values in the left table without going under the left value. For example, if the right table contains a value `5` and the left table contains values `4` and `6`, the right table's `5` will be matched on the left table's `4`.
```

> Let's also join these tables using a `raj`.

Reviewer: ```suggestion
Let's also join these tables with the `raj` method.
```

```python
joined_data_raj = daily_data_0.raj(daily_data_1, "Timestamp", "Number1 = Number, Character1 = Character, Boolean1 = Boolean")
```

> As of joins work very well with time-tables that sample at different frequencies. Let's create two new tables, one that samples every second and one that samples every ten seconds, and show what happesn when we join them together using `aj` and `raj`.

Reviewer: ```suggestion
As-of-joins work very well with time tables that sample at different frequencies. Let's create two new tables, one that samples every second and one that samples every ten seconds, and show what happens when we join them together using `aj` and `raj`.
```
